templates/release.mako

```diff
@@ -52,23 +72,16 @@ You can use freely available tools like [url=http://www.cpuid.com/softwares/cpu-
 [img]http://scp.indiegames.us/img/mac-icon.png[/img] [color=green][size=12pt]OS X Universal (32/64-bit Intel)[/size][/color]
 [size=8pt]Compiled on OS X 10.11.4, Xcode 7.3 ([url=https://gist.github.com/yamaya/2924292]Apple LLVM version cross-reference[/url])[/size]
 
-[url=http://scp.fsmods.net/builds/OSX/fs2_open_3.7.4_RC2.dmg]fs2_open_3.7.4_RC2.dmg[/url] ([url=http://scp.indiegames.us/builds/OSX/fs2_open_3.7.4_RC2.dmg]Mirror[/url]) ([color=yellow][url=http://scp.fsmods.net/builds/OSX/fs2_open_3.7.4_RC2.md5]MD5[/url][/color])
-
-
-[hidden=Other Platforms, Source Code]
-[img]http://scp.indiegames.us/img/linux-icon.png[/img] [color=green][size=12pt]Ubuntu Linux 32-bit[/size][/color]
-[size=8pt]Compiled on Ubuntu 14.04.4 LTS 32-bit, GCC 4.8.4[/size]
-The Ubuntu builds are built and provided as a reference, and also because Ubuntu is a very common Desktop Linux distribution.  We usually recommend Linux users compile their own builds, but if you happen to be running 32-bit Ubuntu, or 64-bit Ubuntu with 32-bit libraries installed, these may work fine for you.  There have been reports of package managers maintaining FSO packages in various Linux distributions, if you plan to package an FSO release for a distribution please let use know and we will include a reference to it in our release posts.
+${build(groups["MacOSX"].mainFile)}
 
-[url=http://scp.fsmods.net/builds/LINUX/fs2_open_3.7.4_RC2.tar.bz2]fs2_open_3.7.4_RC2.tar.bz2[/url] ([url=http://scp.indiegames.us/builds/LINUX/fs2_open_3.7.4_RC2.tar.bz2]Mirror[/url]) ([color=yellow][url=http://scp.fsmods.net/builds/LINUX/fs2_open_3.7.4_RC2.md5]MD5[/url][/color])
+[img]http://scp.indiegames.us/img/linux-icon.png[/img] [color=green][size=12pt]Linux 64-bit[/size][/color]
+[size=8pt]Compiled on Ubuntu 14.04.4 LTS 64-bit, GCC 5[/size]
```

Comment:
That should match what Travis is using. I made sure to keep that part valid when I wrote the template.